src/PaymentDetails.res

```diff
@@ -97,4 +97,9 @@ let details = [
     icon: Some(icon("bank", ~size=21)),
     displayName: "BECS Debit",
   },
+  {
+    type_: "boleto",
+    icon: Some(icon("ach", ~size=19)),
+    displayName: "Boleto",
+  },
```

Comment:
is there a separate icon for this ? you can ask designers

Reply:
@prafulkoppalkar have added it in orca.svg

Reply:
but it will still pick it up with id "ach"

Reply:
Resolved now!